extensions/admin-tool-ext-win/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "admin-tool-ext-win",
 	"displayName": "%adminToolExtWin.displayName%",
 	"description": "%adminToolExtWin.description%",
-	"version": "0.0.1",
+	"version": "0.0.2",
```

Comment:
out of curiosity it looks like the other extensions are versioned as "0.<num>.0" Any reason why the admin tool extension is special?

Reply:
Because I didn't look at the other extensions when I made this =P I figure when we GA this extension we'll bump it up to 1.0.0 or 0.1.0 so haven't really been concerned about it up until now